grease-cli/tests/test.lua

```diff
@@ -14,3 +14,17 @@ function no_heuristic() check 'Unable to find a heuristic for any goal' end
 function req_failed() check 'One or more assertions failed' end
 
 function uninit_stack() check 'Likely bug: uninitialized stack read' end
+
+-- TODO: Upstream to Oughta
```

Comment:
See https://github.com/GaloisInc/oughta/pull/25

I would be happy to cut a release and bump our dependency if we'd like, or to create a numbered TODO to remove this when we do so.

Reply:
I'm fine with either choice.

Reply:
https://github.com/GaloisInc/oughta/pull/26